synapse/lib/storm.lark

```diff
@@ -178,7 +185,16 @@ embedquery: _EMBEDQUERYSTART query "}"
 _EMBEDQUERYSTART: "${"
 
 // An unquoted storm string constant.  Must start with a word char, ?, or - and not contain certain punctutation
-NONQUOTEWORD: /[\w\-?][^ \t\n),=\]}|]*/
+NONQUOTEWORD: /[\w\-\+\?\*][^ \t\n),=\]}|]*/
+
+// An unquoted string within a whitespace list
+WORDTOKN: /[\w\+\-\?\*\/\\][^ \t\r\n\=\|\}\)]*/
+
+// An unquoted string within a list syntax
+LISTTOKN: /[\w\+\-\?\*\/\\]([^,)]*[^\s,)])*/
+
+// A white-space *only* delimited string immediate for use in legacy cmdr parser
+CMDRTOKN: /[^\s\{}]+/
```

Comment:
We want to disallow equals, double quote, and single quote here too, at least to start the token.